units/crons.go

```diff
@@ -842,7 +842,7 @@ func PopulateHostCreationJobs(env evergreen.Environment, part int) amboy.QueueOp
 			if h.UserHost || h.SpawnOptions.SpawnedByTask {
 				// pass:
 				//    always start spawn hosts asap
-			} else if submitted > 32 {
+			} else if submitted > env.Settings().HostInit.HostThrottle {
```

Comment:
does .Settings() returned cached? I'm never clear on this.

Reply:
its definitely cached, so to impact this you'd need to restart.

Reply:
Is there a way to get fresh settings?

Reply:
I think
https://github.com/evergreen-ci/evergreen/blob/7fd4857279b6263d3ae5b56d028777d2c94da5d2/config_hostinit.go#L17
will refresh from the db